0.download_data/README.md

```diff
@@ -9,15 +9,12 @@ Complete instructions for data download and preprocessing can be found at: https
 ### Usage
 
 In this repository, all labeled data is downloaded from a version controlled [mitocheck_data](https://github.com/WayScience/mitocheck_data).
-
-The version of mitocheck_data used is specified by the hash corresponding to a current commit.
-The current hash being used is `e1f86cd007657f8247310b78df92891b22e51621` which corresponds to [mitocheck_data/e1f86cd](https://github.com/WayScience/mitocheck_data/tree/e1f86cd007657f8247310b78df92891b22e51621).
-The `hash` variable can be set in [download_data.ipynb](download_data.ipynb) to change which version of mitocheck_data is being accessed.
+We specify the path to each set of `mitocheck_data` with `labeled_data_paths` in [download_data.ipynb](download_data.ipynb).
 
 ### Data Preview
 
 The labeled dataset includes CellProfiler (CP) and DeepProfiler (DP) features as well as metadata (location, perturbation, etc) for cells from the original MitoCheck project.
-The breakdown of cell counts by phenotypic class (as labeled manually by MitoCheck) is as follows:
+The breakdown of cell counts by manually-assigned phenotypic class for the `ic` (illumination corrected) dataset is as follows:
```

Comment:
why is it necessary to specify for the IC dataset? Isn't it the same for no-ic?

Reply:
I believe they differ (very slightly). Because illumination correction has a very small effect on segmentation, we get different cell locations and in some cases a different number of cells.

I was going to update this when I update the path that we download the `no_ic` data from in https://github.com/WayScience/mitocheck_data/pull/37